### DISCUSSION TITLE
Strimzi/Kubernetes/OpenShift compatibility matrix

### DISCUSSION THREAD

**Discussion by antonioproietti** [strimzi/strimzi-kafka-operator#7696] [Q&A] [answered]
Hi all, 
at one of our customers we inherited an old setup of strimzi, version 0.17.0 on Openshift 3.11 (out of support from RedHat, I know...)
We tried to expose kafka outside the cluster using route following the old 0.17.0 documentation, but without success so far.
Seeing that the latest version of Strimzi requires at least kubernetes 1.19, is there a compatibility matrix somewhere?
Can anyone confirm that 0.17.0 is supported on k8s 1.11 (kubernetes version on OCP 3.11) or what is the latest Strimzi version that can run on OCP 3.11?
Thanks and regards,

**Comment by scholzj** [comment, marked as answer]
There is no matrix, but the known versions are listed in the downloads page: https://strimzi.io/downloads/ ... OCP 3.11 should be _supported_ up until 0.20 release (including).

**Reply by antonioproietti** [reply]
Thanks a lot for the quick reply. Sure there is some misconfiguration in route exposure on openshift, so we focus on that problem rather than a version upgrade. Thanks again.
